src/utils/regexp.js

```diff
@@ -0,0 +1,7 @@
+export const checkToOnlyNumber = function (value) {
+  return /^-?\d*$/.test(value);
+};
+
+export const moneyFormat = function (value) {
+  return value.replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+};
```

Comment:
function checkToOnlyNumber for check input for type number, moneyFormat for format money